paltax/source_models_test.py

```diff
@@ -33,10 +34,12 @@
 from paltax import source_models
 from paltax import utils
 
+# NOTE: The tests are done assuming that the test
```

Comment:
I like the inclusion of this note, but put it closer to where the tests make this assumption instead of the top of the file.